moveit_ros/planning/trajectory_execution_manager/src/trajectory_execution_manager.cpp

```diff
@@ -817,7 +817,10 @@ bool TrajectoryExecutionManager::configure(TrajectoryExecutionContext& context,
                                            const moveit_msgs::RobotTrajectory& trajectory,
                                            const std::vector<std::string>& controllers)
 {
-  if (trajectory.multi_dof_joint_trajectory.points.empty() && trajectory.joint_trajectory.points.empty())
+  if (trajectory.multi_dof_joint_trajectory.points.empty() &&
+      (trajectory.joint_trajectory.points.empty() ||
+       trajectory.joint_trajectory.header.stamp + trajectory.joint_trajectory.points.back().time_from_start ==
+           ros::Time()))
```

Comment:
as there's now three conditions, one of which takes a couple of seconds to 'parse': perhaps add a comment explaining what this is checking for / guarding against?